addons/mail/static/src/thread/thread_service.js

```diff
@@ -0,0 +1,35 @@
+/** @odoo-module */
```

Comment:
I am not sure I understand why we need a thread service?

Reply:
Put everything inside messaging class feel like the file will became quickly a bug pile of what ever functions . It's already +500 LoC.
Maybe the thread service is not needed/not the solution but I think we do need a way to split stuff